contracts/osmo-liquid-pooler/src/contract.rs

```diff
@@ -101,14 +102,13 @@ pub fn instantiate(
     IBC_CONFIG.save(deps.storage, &ibc_config)?;
 
     Ok(Response::default()
-        .add_message(enqueue_msg(clock_addr.as_str())?)
```

Comment:
Hey @bekauz would like your review before merging, specially this line that I removed here. My understanding is that this contract is already queued in the clock contract when instantiating the covenant, but then I'm having trouble understanding how was this actually working before if this would fail because it was already queued.

Reply:
Before the current refactoring and introduction of Contract operation mode, including the `initial_queue` field in the Clock's instantiation message, messages had to enqueue themselves in the clock, which was only allowing whitelisted addresses to submit the Enqueue message. You can check the [code](https://github.com/timewave-computer/covenants/tree/v0.1.0) of the official v0.1.0 release.